src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,49 @@
 import React from 'react';
+import './PostInfo.scss';
+import { Posts } from '../../types/Post';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type Props = {
+  post: Posts,
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title,
+    body,
+    user,
+    comments,
+  } = post;
+
+  return (
+    <li className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">
+          {title}
+        </h3>
+
+        {user && (
+          <p>
+            {' Posted by  '}
+            <UserInfo
+              user={user}
+            />
```

Comment:
wrapping if you have more that 2 attributes or your srt length more than 80
```suggestion
            <UserInfo user={user} />
```

Reply:
I know, i thought it will be better for reading.
Okm l'll change it.
Thanks